src/Enemy.cpp

```diff
@@ -100,12 +100,16 @@ void Enemy::areaPatrol(float deltaTime)
 	if (patrol_pause > 0)
 	{
 		if (getX() >= patrol_origin + PATROL_RADIUS)
+		{
 			if (patrol_right)//on the right edge of patrol boundary
 				patrol_right = false;
+		}
 
 		else if (getX() <= patrol_origin - PATROL_RADIUS)
```

Comment:
This else was getting triggered on the inner `if` above rather than the intended outer one.  Fix this by adding braces.